lib/libutils/ext/arch/arm/sub.mk

```diff
@@ -1,3 +1,5 @@
 srcs-$(CFG_ARM32_$(sm)) += aeabi_unwind.c
 srcs-$(CFG_ARM32_$(sm)) += atomic_a32.S
 srcs-$(CFG_ARM64_$(sm)) += atomic_a64.S
+srcs-$(CFG_ARM32_$(sm)) += mcount_a32.S
+srcs-$(CFG_ARM64_$(sm)) += mcount_a64.S
```

Comment:
```
ifneq ($(sm),core)
srcs-$(CFG_ARM32_$(sm)) += mcount_a32.S
srcs-$(CFG_ARM64_$(sm)) += mcount_a64.S
endif
```

Reply:
This would be a redundant change which will be removed in subsequent patch. But its minimal change to have more granular patch-set.